src/utils/components/Consoles/components/CloudInitCredentials/InlineCodeClipboardCopy.tsx

```diff
@@ -3,21 +3,32 @@ import React, { FC } from 'react';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { ClipboardCopy } from '@patternfly/react-core';
 
-import { clipboardCopyFunc } from '../../utils/utils';
+type InlineCodeClipboardCopyProps = {
+  clipboardText: string;
+  hideText?: boolean;
+};
 
-const InlineCodeClipboardCopy: FC<{ clipboardText: string }> = ({ clipboardText }) => {
+const InlineCodeClipboardCopy: FC<InlineCodeClipboardCopyProps> = ({
```

Comment:
it seems we have a PatternFly component with similar scope - https://www.patternfly.org/components/clipboard-copy/#inline-compact-with-additional-action
What features are missing there ?